events/api.py

```diff
@@ -1306,14 +1315,36 @@ def _filter_event_queryset(queryset, params, srs=None):
         dt = parse_time(val, is_start=False)
         queryset = queryset.filter(Q(last_modified_time__gte=dt))
 
-    val = params.get('start', None)
-    if val:
-        dt = parse_time(val, is_start=True)
+    if params.get('format') == 'docx':
+        if not params.get('location'):
+            raise ValidationError(_('Must specify a location.'))
+
+    start = params.get('start')
+    end = params.get('end')
+    days = params.get('days')
+
+    if days:
+        try:
+            days = int(days)
+        except ValueError as e:
+            raise ParseError(_('Error while parsing days.'))
+        if days < 1:
+            raise ParseError(_('Days must be 1 or more.'))
+
+        if start or end:
+            raise ValidationError(_('Start or end cannot be used with days.'))
+
+        today = datetime.now(timezone.utc).date()
+
+        start = today.isoformat()
```

Comment:
Because the days parameter simulates the get parameter that would be a isoformat date, so it's either complicating the later logic (by checking if the days parameter was set) or creating a 1.2 millisecond (yes I measured :smiley: ) delay per request.